chain/bitcoind_events.go

```diff
@@ -21,10 +37,29 @@ type BitcoindEvents interface {
 
 // NewBitcoindEventSubscriber initialises a new BitcoinEvents object impl
 // depending on the config passed.
-func NewBitcoindEventSubscriber(cfg *BitcoindConfig,
+func NewBitcoindEventSubscriber(cfg *BitcoindConfig, client *rpcclient.Client,
 	notifyBlock func(block *wire.MsgBlock),
 	notifyTx func(tx *wire.MsgTx)) (BitcoindEvents, error) {
 
+	if cfg.PollingConfig != nil && cfg.PollingConfig.Enable {
```

Comment:
Add a sanity check to ensure that _both_ the polling and zmq configs aren't populated?